trac_configurable_ctu/__init__.py

```diff
@@ -93,6 +113,19 @@ def expand_macro(self, formatter, name, content, args=None):
         except Exception: # pylint: disable=broad-except
             message = content
             resource = Resource('repository', reponame)
+        config = self.ticket_replace_section
+        fields = {}
+        for key, value in config.options():
+            prefix, option = key.split('.')
```

Comment:
You should probably make this `key.split('.', 1)` to prevent the code from crashing with `name.pattern.broken` inputs.